### DISCUSSION TITLE
Controlling a CF drone with OptiTrack MoCap system. Drone starts spinning around uncontrollably.

### DISCUSSION THREAD

**Discussion by noahra** [bitcraze/discussions#789] [Q&A]
I am running ROS Noetic on a Docker container and I am running my Crazyswarm script from that container. 

**After flying the drone for a while, it starts to spin uncontrollably.** 

I am using a OptiTrack MoCap-system. 

I have no Flow Deck.

After taking off I get this message

` [ WARN] [1687261863.372791337]: No updated pose for CF cf1 for 6.433032 s.`

**Comment by krichardsson** [comment]
I'm not sure, but I guess you are using Crazyswarm? If that is the case, please head over to the Crazyswarm support, they will be better at helping you. You can find them here: [Crazyswarm](https://github.com/USC-ACTLab/crazyswarm/discussions) and [Crazyswarm2](https://github.com/IMRCLab/crazyswarm2/discussions)

If this is not the case, please describe your setup a bit more in detail.

**Reply by noahra** [reply]
I am using Crazyswarm. I'll make sure to look there.

I am suspecting it has to do with choosing PID as controller but not having a flow deck.
